tests/test_transpilers_abstract.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from qibo import gates
+from qibo.models import Circuit
+
+from qibolab.transpilers.abstract import create_circuit_repr
+
+
+def test_circuit_representation():
+    circuit = Circuit(5)
+    circuit.add(gates.CNOT(1, 0))
+    circuit.add(gates.CNOT(2, 0))
+    circuit.add(gates.X(1))
+    circuit.add(gates.CZ(3, 0))
+    circuit.add(gates.CNOT(4, 0))
+    repr = create_circuit_repr(circuit)
+    print(repr)
```

Comment:
Remove all `print`s before merging.

- if you want to keep in the package, log instead of printing
- in tests, they are only for debug, so just drop them